### PR TITLE
feat(ui-ux): update transaction details page

### PR DESCRIPTION
**Pull request by lykalabrada**
#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [ ] Read your code changes at least once
- [ ] Your UI implementation visually matched the rendered design\*
- [ ] Unit tests\*
- [ ] Added e2e tests\*

### CODE REVIEW

File: src/pages/tx/_components/GasDetails.tsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
+import useWindowDimensions from "hooks/useWindowDimensions";
```
Comment (pierregee) [inline src/pages/tx/_components/GasDetails.tsx:1]: Spacing between figma design and actual on gas detail section (desktop view)
image
image

File: src/pages/tx/[tid].tsx [hunk 2]
```diff
@@ -1,1114 +1,261 @@
...
+
+  const titleFontCss =
+    "text-white-700 break-all leading-[22.4px] tracking-[0.01em] lg:leading-[24px] lg:-tracking-[0.02em]";
+  const valueFontCss = "text-white-50 leading-[24px] -tracking-[0.02em]";
+  const rowCss =
+    "flex md:flex-col justify-between md:justify-normal gap-y-2 h-6 md:h-auto";
 
   return (
-    <div className="px-1 md:px-0 mt-12">
+    <div className="px-1 md:px-0 mt-12 antialiased">
```
Comment (pierregee) [inline src/pages/tx/[tid].tsx:41]: image
Very responsive! one minor thing, can add minimum width to handle this

**Review by pierregee — CHANGES_REQUESTED** [review]

File: src/pages/tx/_components/GasDetails.tsx (hunk 1 above)
Comment (lykalabrada) [reply src/pages/tx/_components/GasDetails.tsx:1]: weird, just checked both on local and deploy preview and it looks fine for me 😵 
image

File: src/pages/tx/[tid].tsx (hunk 2 above)
Comment (lykalabrada) [reply src/pages/tx/[tid].tsx:41]: Updated! Thanks for catching!

**Review by pierregee — APPROVED** [review]

**Pull request merged**